src/themes/components/PostDate/index.js

```diff
@@ -1,16 +1,27 @@
 /* globals window */
 const ComponentThemes = window.ComponentThemes;
-const { React, registerComponent } = ComponentThemes;
+const { React, registerComponent, apiDataWrapper } = ComponentThemes;
+
+const PostDate = ( { date, date_format, className } ) => {
+	var dateStr = '';
+
+	if ( date ) {
+		dateStr = ComponentThemes.date.format( date_format, new Date( date ) );
+	}
 
-const PostDate = ( { date, className } ) => {
 	return (
 		<span className={ className }>
-			{ date || 'No date' }
+			{ dateStr || 'No date' }
 		</span>
 	);
 };
 
-registerComponent( 'PostDate', PostDate, {
+const mapApiToProps = ( getApiEndpoint ) => {
+	const settings = getApiEndpoint( '/wp/v2/settings' ) || [];
```

Comment:
This happens to me when going back and forth from PHP to JS, but you probably mean `... || {}` here and not `... || []`